browser/react/index.js

```diff
@@ -47,7 +47,11 @@ ReactDOM.render(
     <MuiThemeProvider>
       <Router history={browserHistory}>
         <Route path="/" onEnter={onHomeEnter} >
-          <IndexRoute component={Login} />
+          <IndexRedirect to="/login" />
```

Comment:
It actually does work. Since 52 and 53 have '/login' and '/signup/', they're not actually nested under home. I don't really think that we need a /home route though, since it's just a blank background, so maybe just delete lines 51 and 54?